src/Pages/ProductDetails/Components/GoodsDetails/Components/GoodsDescription/GoodsDescription.js

```diff
@@ -0,0 +1,78 @@
+import Taps from 'Pages/ProductDetails/Components/Taps/Taps';
+import React, { Component } from 'react';
+import './GoodsDescription.scss';
+class GoodsDescription extends Component {
+  state = {
+    info: [],
+  };
+  componentDidMount() {
```

Comment:
```suggestion
import './GoodsDescription.scss';

class GoodsDescription extends Component {
  state = {
    info: [],
  };
  
  componentDidMount() {
```
내용상 구분되는 요소들 사이에는 한줄 띄워 가독성 확보해주세요 👀